src/transformers/models/auto/tokenization_auto.py

```diff
@@ -232,7 +234,6 @@
         (MarianConfig, (MarianTokenizer, None)),
         (BlenderbotSmallConfig, (BlenderbotSmallTokenizer, None)),
         (BlenderbotConfig, (BlenderbotTokenizer, None)),
-        (LongformerConfig, (LongformerTokenizer, LongformerTokenizerFast)),
```

Comment:
Yes it can, it's included twice on master:
![image](https://user-images.githubusercontent.com/48327001/115046994-60daab00-9ed8-11eb-91e5-c9245246f34e.png)

Reply:
Indeed, great find!